server.go

```diff
@@ -54,7 +54,8 @@ func (s *server) Serve() chan error {
 	ec := make(chan error)
 
 	mux := http.NewServeMux()
-	mux.HandleFunc("/multihash/", s.find)
+	mux.HandleFunc("/cid/", s.find)
+	mux.HandleFunc("/multihash", s.find)
```

Comment:
will this also handle `/multihash/*`? this is not a gorilla mux, just the golang one, and i think we need both